src/main/java/com/github/mauricioaniche/ck/util/MethodCounter.java

```diff
@@ -0,0 +1,77 @@
+package com.github.mauricioaniche.ck.util;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class MethodCounter {
```

Comment:
The string produced by this class can be _really_ long. I'm not sure I'd like to mix the CSV, which is mostly numbers with this super large string. Also, given that this string can be quite long, just concatenating it to the CSV might make the CSV invalid. We'd need to use proper escaping here. An alternative would be to output this to a separate file. And make this feature disabled by default, so we don't start producing large data files to people that have been using CK so far (and do not expect it).